Source/Generators/LMCArraySignalGenerator.cc

```diff
@@ -412,7 +412,7 @@ namespace locust
         const double elementRadius = fArrayRadius;
         double zPosition;
         double theta;
-        const double dThetaArray = 2. * LMCConst::Pi() / nChannels; //Divide the circle into nChannels
+        const double dThetaArray = 2. * LMCConst::Pi() / (nChannels/nSubarrays); //Divide the circle into nChannels

```

Comment:
Is this right ? From the code it looks like the total number recievers(patches/slots) = nsubarray*nchannels*nelementsPerchannel. But, this line seems to divide the circumference of array by number of channels and multiply by the number of arrays.